packages/runtime/runtime-utils/src/serializer.ts

```diff
@@ -9,12 +9,61 @@
 import {
     IFluidHandle,
     IFluidHandleContext,
-    IFluidSerializer,
 } from "@fluidframework/core-interfaces";
 import { RemoteFluidObjectHandle } from "./remoteObjectHandle";
 import { generateHandleContextPath } from "./dataStoreHandleContextUtils";
 import { isSerializedHandle } from "./utils";
 
+/**
+ * JSON serialized form of an IFluidHandle
+ */
+ export interface ISerializedHandle {
+    // Marker to indicate to JSON.parse that the object is a Fluid handle
+    type: "__fluid_handle__";
+
+    // URL to the object. Relative URLs are relative to the handle context passed to the stringify.
+    url: string;
+}
+
+export const IFluidSerializer: keyof IProvideFluidSerializer = "IFluidSerializer";
+
+export interface IProvideFluidSerializer {
```

Comment:
Do we need IProvideFluidSerializer? I suspect we don't.

Reply:
I do not see any usage, so the answer should be no - we should delete it.